pdr_backend/pdr_dashboard/test/conftest.py

```diff
@@ -95,7 +99,10 @@ def _add_css(app):
 
 @pytest.fixture
 def setup_app(
-    tmpdir, _sample_daily_predictions, _sample_payouts_related_with_predictions
+    tmpdir,
+    _sample_daily_predictions,
+    _sample_payouts_related_with_predictions,
+    _sample_subscriptions,
```

Comment:
hey guys, this is not kosher

if I need to change the schema to provide support, your dashboards will start breaking because of this and I will have to spend even more of my time to fix your code

I have asked several times now to address this
https://oceanprotocol.slack.com/archives/C066NE56MF0/p1722625321763779

please use the latest fixtures and delete the code for mock_subscriptions, mock_predictions etc...

@KatunaNorbert @calina-c @kdetry @trentmc

Reply:
handling it inside #1522 for the Feeds table, then will handle it for the other parts inside a different issue